phoenicis-tests/src/main/java/com/playonlinux/tests/PhoenicisTestsApp.java

```diff
@@ -0,0 +1,69 @@
+package com.playonlinux.tests;
+
+import com.playonlinux.apps.ApplicationsSource;
+import com.playonlinux.apps.dto.ApplicationDTO;
+import com.playonlinux.apps.dto.CategoryDTO;
+import com.playonlinux.apps.dto.ScriptDTO;
+import com.playonlinux.multithreading.ControlledThreadPoolExecutorServiceCloser;
+import com.playonlinux.scripts.interpreter.ScriptInterpreter;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PhoenicisTestsApp {
```

Comment:
I think it would be better to have several integration tests instead of this PhoenicisTestApp. 

E.g. OnlineInstallerScriptTest: 
1. let the ApplicationSourceMock return a script which installs without user interaction
2. run the installation
3. check the result with JUnit means